h2o-jetty-8/src/main/java/water/webserver/jetty8/Jetty8ServerAdapter.java

```diff
@@ -134,9 +134,10 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
 
   private class GateHandler extends AbstractHandler {
     @Override
-    public void handle(String target, Request baseRequest, HttpServletRequest request, HttpServletResponse response) {
-      final boolean handled = h2oHttpView.gateHandler(request, response);
-      if (handled) {
+    public void handle(String target, Request baseRequest, HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Jetty 8 module should be gone soon. For now, I suggest we ignore the code duplication over here.

Reply:
why not put this into the `boolean gateHandler` method? would avoid the duplication

Reply:
That would avoid a lot of things actually. Let me see.

Reply:
That actually led to a lot of simplifications, as no additional state transfer to webserver impl from core is done. Everything stays "nicely" in core.